linux/build.sh

```diff
@@ -30,3 +30,12 @@ builder_describe \
 builder_parse "$@"
 
 builder_run_child_actions clean configure build test install uninstall
+
+test_action() {
+  if builder_has_option --report; then
+    builder_echo "Opening coverage reports in browser..."
+    xdg-open "file://${THIS_SCRIPT_PATH}/CodeCoverageReports.html"
+  fi
```

Comment:
Does this create a coverage report or open it in the browser or both? It would be better for the creation of the report to be separated from the opening of it for use in CI.

Reply:
I added a `--open` option to open the report.